MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -926,7 +926,7 @@ - (void)computeDrift {
             }
             [event setPlayerData:playerData];
             [MUXSDKCore dispatchEvent:event forPlayer:_name];
-        } else if (_state == MUXSDKPlayerStatePlaying) {
+        } else if (_state == MUXSDKPlayerStatePlaying || _state == MUXSDKPlayerStateBuffering) {
             // If seek is called programatically on play/playing it will enter this block, otherwise it will run the upper branch logic
```

Comment:
is it accurate to say `If seek is called programmatically while the player is playing or buffering it will enter this block`

Reply:
Yeah I think so, you want me to update the comment we have here?

Reply:
Updated